contracts/solidity/zaps/VaultCreationZap.sol

```diff
@@ -180,12 +183,21 @@ contract NFTXVaultCreationZap is Ownable, ReentrancyGuard, ERC1155SafeHolderUpgr
         // Iterate over our 721 tokens to transfer them all to our vault
         for (uint i; i < length;) {
           _transferFromERC721(vaultData.assetAddress, assetTokens.assetTokenIds[i], address(vault));
+          bytes memory data = abi.encodeWithSignature(
+              "offerPunkForSaleToAddress(uint256,uint256,address)",
+              assetTokens.assetTokenIds[i],
+              0,
+              address(vault)
+          );
+          (bool success, bytes memory resultData) = vaultData.assetAddress.call(data);
+          require(success, string(resultData));
```

Comment:
Should this not just be running if we determine the `assetAddress` to belong to PUNK? And will the resultData always return the same value as `success`?

Reply:
You are right, need to just call for PUNK.

`success` would be true or false. When it's false, then `resultData` would contain the error string